### PR TITLE
Use ha-blueprint

### PR DESCRIPTION
**Pull request by KTibow**
I recently announced ha-blueprint:
https://community.home-assistant.io/t/made-a-custom-integration-or-card-ive-created-an-all-in-one-github-action-for-you/235041?u=ktibow
This makes CI use it.

### CODE REVIEW

**Comment by alandtse** [comment]
Thanks. Can you explain what ha-bluepoint does? Is it just putting all the validation into a github action?

**Comment by KTibow** [comment]
Basically yeah, it runs hassfest, HACS validation, and Python validation with flake8

**Comment by alandtse** [comment]
I just saw you #985, what's the difference?

**Comment by KTibow** [comment]
I'm so dumb I made a duplicate PR. Sorry.

**Comment by KTibow** [comment]
Just wondering, is there anything I could tell you or help you with?

**Comment by alandtse** [comment]
We haven't done any releases using the CI so haven't bothered to put this in. Can you show me some projects using this Action? I'm not sure what the benefit is of moving to an action instead of what we already have where it's all independent steps.

**Comment by KTibow** [comment]
The benefit is that it has lot of stuff I've added, even though it looks like the same. Here's some projects using it:
https://github.com/search?q=ha-blueprint+-user%3AKTibow&type=Code&ref=advsearch&l=&l=

**Comment by alandtse** [comment]
Can you let me know if the problems mentioned in the HA community thread are resolved? I also don't get understand your last post. Did GitHub disable your action?

**Comment by KTibow** [comment]
As for the last post: You know how you have to make a fork to make a PR? I had made a lot of forks, and GitHub was lowering down how much load they had to have by disabling GH Actions they thought that didn't need.
Some repos have the issue mentioned in the forums. This one doesn't though, as you can see:
image

**Comment by alandtse** [comment]
Please note I had to switch to HACS Action. Does your action use that or the other check? https://github.com/marketplace/actions/hacs-action

File: .github/workflows/validate.yaml [hunk 1]
```diff
@@ -1,65 +1,30 @@
...
+          path: |
+            ~/.cache/pip
+          key: custom-component-ci
+      - uses: hacs/integration/action@main
         with:
-          python-version: 3.8
-      - name: Cache
-        uses: actions/cache@v2
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          CATEGORY: integration
```
Comment (KTibow) [inline .github/workflows/validate.yaml]:
```suggestion
          CATEGORY: integration
```

File: .github/workflows/validate.yaml [hunk 2]
```diff
@@ -1,65 +1,30 @@
...
-        uses: actions/setup-python@v2
+      - uses: actions/setup-python@v2
+        name: Setup Python
+      - uses: actions/cache@v2
+        name: Cache
+        with:
+          path: |
+            ~/.cache/pip
+          key: custom-component-ci
+      - uses: hacs/integration/action@main
```
Comment (KTibow) [inline .github/workflows/validate.yaml]:
```suggestion
      - uses: hacs/action@main
```

**Review by KTibow — COMMENTED** [review]
Using HACS

**Pull request merged**